docs/source/assets/js/all.js

```diff
@@ -2,5 +2,31 @@
 window.onload = function () {
   // Initialize all calcite.js patterns
   console.log('Initializing calcite!');
+
+  var viewTogglers = window.calcite.nodeListToArray(document.querySelectorAll('.js-view-toggle'));
+
+  viewTogglers.map(function (toggle) {
+    window.calcite.addEvent(toggle, 'click', clickHandler);
+  });
+
+  window.calcite.bus.on('view:toggle', setView);
+
+  function clickHandler (e) {
+    e.preventDefault();
+    var set = e.target.getAttribute('data-toggle-set');
+    var target = e.target.getAttribute('data-toggle');
+    window.calcite.bus.emit('view:toggle', set, target);
+  }
+
+  function setView (set, target) {
```

Comment:
This is some rough code that enabled interactive patterns outside of the calcite core. This is getting moved to proper extension patterns, ultimately.